### ISSUE TITLE
Tracking Protection toggle in the menu should block and unblock JavaScript and Web fonts

### ISSUE THREAD

**Issue by brampitoyo** [mozilla-mobile/focus-android#2368]
Follow-up issue from #907 – thanks for the clarification, @ekager.

When JavaScript and Web fonts are blocked from inside Settings, toggling the main Tracking Protection option off – the one inside the menu – should also unblock them.

This means that the user will be able to fix their site behaviour/visual problems just by opening the menu and tapping the toggle, without needing to go back inside Settings.

**Comment by ekager** [comment]
Does this need more hashing out/strings, or is it a simple: If the menu toggle goes from ON -> OFF, also toggle Blocking JavaScript and Blocking Web Fonts to OFF?

**Comment by brampitoyo** [comment]
@ekager This doesn’t need any more string or explanation. When the menu toggle goes from ON -> OFF, JavaScript and web fonts should also be toggled OFF.

Doin gthis fulfills the user’s expectation that “When I turn this off, pages should open like it would in any other browser. Any problem with displaying the page may be fixed then.”
